pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -1149,3 +1156,40 @@ func PrivilegedContext() *v1.SecurityContext {
 		Privileged: &privileged,
 	}
 }
+
+func (c *Cluster) applyOSDNetworkSettings(osdID string) error {
+	monstore := opconfig.GetMonStore(c.context, c.Namespace)
+	who := fmt.Sprintf("osd.%s", osdID)
+	netSelectors := []string{"public", "cluster"}
+
+	logger.Debugf("public selector is %q and cluster selector is %q", c.Network.Selectors[netSelectors[0]], c.Network.Selectors[netSelectors[1]])
```

Comment:
Looks like this requires both the `public` and `cluster` selectors, which is different from earlier comments. Won't this crash if the selectors aren't specified?

Reply:
Oops!